src/labels_by_user_input.py

```diff
@@ -1,21 +1,48 @@
+import re
+
 from src.utils import get_labels
 
 
-LABEL_VERIFIED = "Verified"
+LABELS_SET = {
+    "hold": "do-not-merge/on-hold",
+    "verified": "Verified",
+    "bug": "bug",
+}
+
+
+def remove_label(pull, label):
+    print(f"Remove {label} from {pull.title}")
+    pull.remove_from_labels(label)
 
 
 def remove_verified_label(pull):
     labels = get_labels(pull=pull)
-    if LABEL_VERIFIED in labels:
-        print(f"Remove {LABEL_VERIFIED} from {pull.title}")
-        pull.remove_from_labels(LABEL_VERIFIED)
+    verified = LABELS_SET["verified"]
+    if verified in labels:
+        remove_label(pull=pull, label=verified)
 
 
 def labels_by_user_input(data, pull):
     body = data["comment"]["body"]
-    if "/verified" in body and LABEL_VERIFIED not in get_labels(pull=pull):
-        print(f"Adding {LABEL_VERIFIED} to {pull.title}")
-        pull.add_to_labels(LABEL_VERIFIED)
+    issue_labels = get_labels(pull=pull)
+    comment_labels = re.findall("/\\S*", body)
+    print(body)
+    print(comment_labels)
+
+    for label in comment_labels:
+        label = label.lstrip("/").lower()
+
+        if "un" in label and label.lstrip("un") in issue_labels:
+            remove_label(pull=pull, label=label.lstrip("un"))
+
+        elif label in LABELS_SET:
```

Comment:
should be checked first, we don;t want to do anything if the /label is not owned by us.